pkg/gatherers/workloads/gather_workloads_info.go

```diff
@@ -79,20 +87,35 @@ func (g *Gatherer) GatherWorkloadInfo(ctx context.Context) ([]record.Record, []e
 		return nil, []error{err}
 	}
 
-	return gatherWorkloadInfo(ctx, gatherKubeClient.CoreV1(), gatherOpenShiftClient)
+	return gatherWorkloadInfo(ctx, gatherKubeClient, gatherOpenShiftClient, imageConfig)
```

Comment:
I shifted a bit the parameters to be able to use the client to connect to the `insights-operator-runtime` pod.